notebooks/work-in-progress/VCERARE.ipynb

```diff
@@ -0,0 +1,288 @@
+    "\n",
+    "```\n",
+    "# Filter by single year\n",
+    "filtered_df = rare_df[rare_df[\"report_year\"]==2023]\n",
+    "\n",
+    "# Filter by multiple years\n",
+    "filtered_df = rare_df[rare_df[\"report_year\"].isin([2019, 2020])]\n",
+    "\n",
+    "# Filter by state\n",
+    "filtered_df = rare_df[rare_df[\"state\"]==\"TX\"]\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "82d0cc7c-3b84-4dc8-b07d-4f2f6de8df79",
+   "metadata": {
+    "jupyter": {
+     "outputs_hidden": true
+    }
+   },
+   "outputs": [],
+   "source": [
+    "filtered_df = rare_df[rare_df[\"state\"]==\"RI\"]\n",
+    "filtered_df.head(10)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1e5c6c8d-8a8e-48be-918d-6a9d1e14741f",
+   "metadata": {},
+   "source": [
+    "### Visualize the data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "cc41d9ee-3f6b-4da6-8450-6f649bef2744",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# random county_id generator:\n",
+    "random_county = random.choice(rare_df[\"county_id_fips\"])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "cb3e5f29-7c98-44a2-8578-b64a6878733d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Editable fields: \n",
+    "year = 2023\n",
+    "county_id = random_county\n",
+    "gen_type = \"onshore_wind\" # choose from: solar_pv, onshore_wind, offshore_wind"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3915c2de-215f-4b25-b718-15f14ed3dbd6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Non-editable variables\n",
+    "plot_df = rare_df[\n",
+    "    (rare_df[\"report_year\"]==year)\n",
+    "    & (rare_df[\"county_id_fips\"]==county_id)\n",
+    "]\n",
+    "county_name = plot_df[plot_df[\"county_id_fips\"]==county_id].county_or_lake_name.unique().item()\n",
+    "state_name = plot_df[plot_df[\"county_id_fips\"]==county_id].state.unique().item()\n",
+    "\n",
+    "# Make the chart\n",
+    "plt.hist(plot_df[f\"capacity_factor_{gen_type}\"], bins=100, range=(0, 1))\n",
+    "plt.title(\n",
+    "    f\"county: {county_name}; state: {state_name}; gen: {gen_type}; max:{plot_df[f\"capacity_factor_{gen_type}\"].max():.2f}\")\n",
+    "plt.grid()\n",
+    "plt.show();"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "09d939e8-3a3d-4908-9ca3-67e4c69c35d8",
+   "metadata": {},
+   "source": [
+    "### Download the data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "13289ced-0028-4d79-9e5a-cf79bc464d80",
+   "metadata": {},
+   "source": [
+    "#### Memory check - will it Excel? \n",
+    "\n",
+    "If you want to know whether your table is capable of being processed as a csv you can use this memory estimator. If the estimated memory exceeds 500 MB it's too big! Cutting columns or making the filter scope smaller will help reduce the file size."
```

Comment:
If we're saying "cutting columns", we should show people how to drop columns probably.